### PR TITLE
New module: digital_ocean_certificate_facts

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/digital_ocean/digital_ocean_certificate_facts.py

##### ANSIBLE VERSION

```
2.6devel
```

### CODE REVIEW

**Comment by Akasurde** [comment]
@pmarques @BondAnthony Could you please review this ?

File: lib/ansible/modules/cloud/digital_ocean/digital_ocean_certificate_facts.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+- name: Gather facts about all certificates
+  digital_ocean_certificate_facts:
+    oauth_token: "{{ oauth_token }}"
+
+- name: Gather facts about certificate with given id
+  digital_ocean_certificate_facts:
+    oauth_token: "{{ oauth_token }}"
+    certificate_id: "892071a0-bb95-49bc-8021-3afd67a210bf"
+
+- name: Get not after facts about certificate
```
Comment (BondAnthony) [inline lib/ansible/modules/cloud/digital_ocean/digital_ocean_certificate_facts.py:47]: Would a better example be the use of the `sha1_fingerprint` or `id`?

Comment (Akasurde) [reply lib/ansible/modules/cloud/digital_ocean/digital_ocean_certificate_facts.py:47]: OK.

Comment (BondAnthony) [reply lib/ansible/modules/cloud/digital_ocean/digital_ocean_certificate_facts.py:47]: I'm just trying to think of a simple use case for these facts.

Comment (Akasurde) [reply lib/ansible/modules/cloud/digital_ocean/digital_ocean_certificate_facts.py:47]: Use case can be 
- identify existing certificates in Digital Ocean
- check expiration of certificates and automation related to that
- check validity of certificates

**Comment by Akasurde** [comment]
@pmarques @BondAnthony Any news ?

**Review by BondAnthony — APPROVED** [review]
Tested and works as expected.

**Comment by BondAnthony** [comment]
shipit

**Review by pmarques — APPROVED** [review]

**Comment by pmarques** [comment]
LGTM

shipit

**Comment by Akasurde** [comment]
@pmarques @BondAnthony Thanks for review.

**Pull request merged**